packages/apiserver/src/data-loader-type.ts

```diff
@@ -0,0 +1,14 @@
+const DataLoaderType = {
```

Comment:
can be named as `DataLoaders` (like previously as we don't use it as type, could be misleading)

Reply:
Renamed to DataLoaders